### ISSUE TITLE
const correctness 

### ISSUE THREAD

**Issue by brucebob** [tinfoilboy/CTML#2]
Hello I like the idea of the library and it's pretty intuitive to use but i notices a few thing about it that doesn't seem right.

``` c++
#include "CTML.h"
#include <iostream>

int main ()
{
    const CTML::Document document;
    std::cout << document.ToString(CTML::MULTILINE);
}
```

This will fail to compile due to ToString not being a const member function. Now unless ToString mutates Document it should be const function.

I also noticed this with your get functions have the same issue.

**Comment by tinfoilboy** [comment]
You're right, all of those functions should be const. I'll fix that really quick.

**Comment by tinfoilboy** [comment]
Just fixed this in d59d087. Closed.

**Comment by brucebob** [comment]
You put the const in the wrong position.

old

``` c++
std::string ToString(const Readability& readability) {
```

how it should look so you can call ToString on a const object

``` c++
std::string ToString(const Readability& readability) const {
```

What you had only make the std::string const not the member function

**Comment by tinfoilboy** [comment]
Oh, that makes sense. Sorry about that!

**Comment by tinfoilboy** [comment]
Okay, now the issue _should_ be fixed in fb8599f.
